### ISSUE TITLE
getObject: ENOTFOUND

### ISSUE THREAD

**Issue by austinmcorso** [Nike-Inc/aws-thin-s3-node#2]
Works:
```
const AWS = require('aws-sdk');
const s3 = new AWS.S3({
    apiVersion: '2006-03-01',
    region: 'us-east-1',
  });
  const params = {
    Bucket: "test-bucket",
    Key: "index.html",
  };
  s3.getObject(params, (err, data) => {...});
```

Error: "getaddrinfo ENOTFOUND s3-us-east-1.amazonaws.com s3-us-east-1.amazonaws.com:443"
```
const S3 = require('aws-thin-s3');
const s3 = new S3({
    apiVersion: '2006-03-01',
    region: 'us-east-1',
  });
  const params = {
    Bucket: "test-bucket",
    Key: "index.html",
  };
  s3.getObject(params, (err, data) => {...});
```

**Comment by kyeotic** [comment]
bba3cf27b8f86e914d50cbbbb524dac489b7adf7 should fix this

published on `0.10.2-beta.1`, install with `npm i aws-thin-s3@beta`

**Comment by austinmcorso** [comment]
Almost. PR opened - [https://github.com/Nike-Inc/aws-thin-s3-node/pull/3](https://github.com/Nike-Inc/aws-thin-s3-node/pull/3)
